src/main/java/com/renatusnetwork/momentum/commands/SquadsCMD.java

```diff
@@ -0,0 +1,270 @@
+				if (n != 2)
+					sendHelp(sender);
+				else if (squad == null)
+					sender.sendMessage(NO_SQUAD_MSG);
+				else if (!SquadsManager.isLeader(player))
+					player.sendMessage(Utils.translate("&cYou are not the squad leader!"));
+				else {
+					PlayerStats targetMember = Momentum.getStatsManager().getByName(args[1]);
+					if (targetMember == null || !SquadsManager.isMember(squad, targetMember))
+						player.sendMessage(Utils.translate("&cThat player is not in the squad!"));
+					else if (targetMember.equals(player))
+						player.sendMessage(Utils.translate("&cYou cannot promote yourself!"));
+					else {
+						squadsManager.promote(targetMember);
+						SquadsManager.notifyMembers(squad, "&9SC &3" + targetMember.getDisplayName() + " &bhas been promoted to squad leader");
+					}
+				}
+
+				break;
+			case "chat":
+				if (squad == null)
+					sender.sendMessage(NO_SQUAD_MSG);
+				else if (n == 1)
+					player.sendMessage(Utils.translate("&3You have toggled &9&lSquad Chat &b" + (squadsManager.toggleSquadChat(player) ? "on" : "off")));
+				else {
+					String msg = String.format("&9SC &3%s &b%s", player.getDisplayName(), String.join(" ", Arrays.copyOfRange(args, 1, n)));
+					squadsManager.sendMessage(player, msg, true);
+				}
+
+				break;
+			case "chatspy":
+				if (!sender.hasPermission("momentum.admin"))
+					sendHelp(sender);
+				else
+					player.sendMessage(Utils.translate("&3You have toggled &9&lSquad ChatSpy &b" + (squadsManager.toggleSquadChatSpy(player) ? "on" : "off")));
+
+				break;
+			case "warp":
+				Level level = player.getLevel();
+
+				if (squad == null)
+					sender.sendMessage(NO_SQUAD_MSG);
+				else if (!SquadsManager.isLeader(player))
+					player.sendMessage(Utils.translate("&cYou are not the squad leader!"));
+				else if (squad.hasWarpCooldown())
+					player.sendMessage(Utils.translate("&cWarp is on cooldown"));
+				else if (player.getPlayer().getWorld().getName().equalsIgnoreCase(Momentum.getSettingsManager().player_submitted_world))
+					player.sendMessage(Utils.translate("&cYou cannot warp to a plot!"));
+				else if (!player.inLevel())
+					player.sendMessage(Utils.translate("&cYou are not in a level!"));
+				else if (level.isAscendance())
+					player.sendMessage(Utils.translate("&cYou cannot warp to ascendance!"));
+				else if (level.isEventLevel() || level.isRaceLevel())
+					player.sendMessage("&cYou cannot warp to a race or event!");
+				else {
+					// notify first so failure messages send to individual players after
+					SquadsManager.notifyMembers(squad, "&9SC &3" + player.getDisplayName() + " &bhas warped to " + level.getFormattedTitle());
+					squadsManager.warpMembers(player);
+
+					squadsManager.notifySqChatSpies(String.format("&3%s &9has warped their party to &3%s", player.getDisplayName(), level.getFormattedTitle()));
+				}
+
+				break;
+			default:
+				sendHelp(sender);
+		}
+
+		return true;
+	}
+
+	private void sendHelp(CommandSender sender) {
```

Comment:
Can we make this help page be more in line with the style of the others so it's consistent?